defaults.yaml

```diff
@@ -1,31 +1,29 @@
-kubernetes-app:
+helm-app:
```

Comment:
why did you change this?

Reply:
IMO streams-bootstrap is a Helm-App. We use Helm charts and Helm CLI to install/unistall it

Reply:
but namespace applies to all kubernetes-apps
imo it's not necessary to change it

Reply:
Okay reverted